tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -0,0 +1,589 @@
+        assert_valid_image_response(response)
+        content = response.content[0].text.lower()
+        # Should mention comparison or differences
+        assert any(
+            word in content for word in COMPARISON_KEYWORDS
+        ), f"Response should contain comparison keywords. Got content: {content}"
+
+    @skip_if_no_api_key("anthropic")
+    def test_10_complex_end2end(self, anthropic_client, test_config):
+        """Test Case 10: Complex end-to-end with conversation, images, and tools"""
+        # Download image for Anthropic format
+        response_img = requests.get(IMAGE_URL)
```

Ruff (0.11.9)

Probable use of `requests` call without timeout

(S113)